src/ess/bifrost/types.py

```diff
@@ -2,4 +2,15 @@
 
 from scipp import Variable
 
+from ess.reduce.nexus import types as reduce_t
+
 ArcNumber = NewType('ArcNumber', Variable)
+
+# See https://github.com/scipp/essreduce/issues/105 about monitor names
```

Comment:
Anything wrong with adding those names?

Reply:
Do you mean adding them to ESSreduce? Do you actually accept those names?